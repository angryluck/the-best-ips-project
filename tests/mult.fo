fun int mult() =
  let n = read(int) in
  let d = read(int) in
  write(n*d)

fun int main() =
  let tmp = mult() in
  let tmp = mult() in
  let tmp = mult() in
  let tmp = mult() in
  let tmp = mult() in
  mult()
