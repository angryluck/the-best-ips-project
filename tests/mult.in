4
6
4
2
9
-6
-7
-1
0
67