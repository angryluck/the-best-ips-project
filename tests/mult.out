24
8
-54
7
0